paymentsheet/src/main/java/com/stripe/android/elements/PaymentElementConfig.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.elements
+
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.model.PaymentSelection
+
+internal data class PaymentElementConfig(
+    val paymentSheetConfig: PaymentSheet.Configuration?,
+    val stripeIntent: StripeIntent,
+    val merchantName: String,
```

Comment:
Is this the same as `paymentSheetConfig.merchantDisplayName`? If yes, can we omit this?